be/dtm0_log.c

```diff
@@ -177,6 +181,23 @@ M0_INTERNAL void dtm0_be_log_destroy_credit(struct m0_be_seg       *seg,
 					    struct m0_be_tx_credit *accum)
 {
 	lrec_be_list_credit(M0_BLO_DESTROY, 1, accum);
+	/* TODO: add entries for the other components of the structure */
+}
+
+static void dtm0_be_log_create_credit(struct m0_be_seg       *seg,
```

Comment:
We have:
```
dtm0_be_alloc_tx_desc_credit()
dtm0_be_alloc_log_rec_credit()
dtm0_be_set_log_rec_credit()
dtm0_be_del_log_rec_credit()
dtm0_be_log_destroy_credit()
dtm0_be_log_create_credit()
```
Can we unify the names here?
Rename the last two functions to:
```
dtm0_be_destroy_log_credit()
dtm0_be_create_log_credit()
```
Or rename the others as, e.g.:
```
dtm0_be_tx_desc_alloc_credit()
```
?

Reply:
Yes, it was a bit messy. I have changed the naming to `<entity>_<verb>_credit`. Also, it caused several cascading changes -- the details are in the commit message (https://github.com/Seagate/cortx-motr/pull/802/commits/d3efcfa00717f443705fe3bf9a6e3d27f5d63085).